proc_psi.go

```diff
@@ -51,19 +51,8 @@ type PSIStats struct {
 	Full *PSILine
 }
 
-// NewPSIStatsForResource reads pressure stall information for the specified
-// resource. At time of writing this can be either "cpu", "memory" or "io".
-func NewPSIStatsForResource(resource string) (PSIStats, error) {
-	fs, err := NewFS(DefaultMountPoint)
-	if err != nil {
-		return PSIStats{}, err
-	}
-
-	return fs.NewPSIStatsForResource(resource)
-}
-
-// NewPSIStatsForResource reads pressure stall information from /proc/pressure/<resource>
-func (fs FS) NewPSIStatsForResource(resource string) (PSIStats, error) {
+// PSIStatsForResource reads pressure stall information from /proc/pressure/<resource>
```

Comment:
You've lost some of the information here about which types of resources are valid, although IMO this should take an enum of some kind instead of a raw string.

Reply:
For now, I'd say restore the missing doc comment and maybe add a TODO in the body here too.